src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -72,6 +73,20 @@ public void MarkBeginOrEndPara()
                 _anyNonWhitespaceSinceLastPara = false;
             }
 
+            public void MarkLineBreak()
+            {
+                // If this is a <br> with nothing before it, then skip it.
+                if (_anyNonWhitespaceSinceLastPara == false)
```

Comment:
do we have test for this change?

Reply:
The entire feature area is a test hole. If you don't see the test in this PR, there are no tests for it.

Reply:
sure, but you could add more tests if you see that's beneficial to make this area more reliable. having test in this area will tell devs what was done intentionally and what is just result of implementation happen to be written in a way.

Reply:
in my experience, whenever there is complex state machine, unless there is tests covering all states, it is hard to know for those missing cases, whether it just happen to be that way or it is done intentionally that way.

for example, it looks like this code and the other code, it looks like multiple line breaks are ignored when used as first elements in quickinfo or when used in the middle of sections. is that intentional for both cases? or it is just the way things were implemented? without test, I dont know.

Reply:
I am not asking you to write tests covering all different states. I was just asking so you think and you decide if you haven't thought about it. if you already did, then move on.

Reply:
I'm happy to add any tests you think of, and am planning to add at least the ones mentioned in comments previously.